server/hotrod/src/main/scala/org/infinispan/server/hotrod/HotRodServer.scala

```diff
@@ -247,7 +248,19 @@ class HotRodServer extends AbstractProtocolServer("HotRod") with Log {
       clientListenerRegistry.setEventMarshaller(Option(marshaller))
    }
 
-   override def stop: Unit = {
+   def addCacheStore(name: String, instance: Object): Unit = {
```

Comment:
Deployed cache stores should be inactive. I can then amend a cache configuration to use that cachestore if needed.